docker-compose.override.yaml

```diff
@@ -1,4 +1,36 @@
 services:
+  ffc-grants-eligibility-checker:
+    build:
+      context: .
+      target: development
+    depends_on:
+      - redis
+    image: ffc-grants-eligibility-checker-development
+    # platform: ${PLATFORM}
+    environment:
+      NODE_ENV: development
+      USE_REDIS: true
+      REDIS_HOSTNAME: redis # Use the Docker service name to connect
+      REDIS_PASSWORD: '' # Add if needed
+      SESSION_CACHE_TTL: 3600000
+
+  ffc-grants-eligibility-checker2:
```

Comment:
What's the purpose of duplicating this?

Reply:
The purpose is so we mimick the envs and have multiple instances for local development/testing. This helps understand the issues on local without troubleshooting the actual environment.